src/parser/druid/guardian/modules/azeritetraits/MasterfulInstincts.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+
+import Analyzer from 'parser/core/Analyzer';
+import SPELLS from 'common/SPELLS';
+import TraitStatisticBox, { STATISTIC_ORDER } from 'interface/others/TraitStatisticBox';
+import { calculateAzeriteEffects } from 'common/stats';
+import { formatPercentage } from 'common/format';
+
+export function masterfulInstinctsStats(combatant) {
+  if (!combatant.hasTrait(SPELLS.MASTERFUL_INSTINCTS.id)) {
+    return 0;
+  }
+
+  const [ mastery, armor ] = combatant.traitsBySpellId[SPELLS.MASTERFUL_INSTINCTS.id]
+    .reduce(([ masteryTotal, armorTotal ], ilevel) => ([
+      (masteryTotal) + calculateAzeriteEffects(SPELLS.MASTERFUL_INSTINCTS.id, ilevel)[0],
+      (armorTotal) + calculateAzeriteEffects(SPELLS.MASTERFUL_INSTINCTS.id, ilevel)[1],
+    ]), [0, 0]);
+
+  return { mastery, armor };
+}
+
+export const STAT_TRACKER = {
+  masterfulInstinctsStats,
+};
+
+class MasterfulInstincts extends Analyzer {
+  mastery = 0;
+  armor = 0;
+
+  constructor(...args) {
+    super(...args);
+    this.active = this.selectedCombatant.hasTrait(SPELLS.MASTERFUL_INSTINCTS.id);
+    const { mastery, armor } = masterfulInstinctsStats(this.selectedCombatant);
+    console.log(mastery, armor);
+    this.mastery = mastery;
```

Comment:
stray console log 

Also should probably return early in the constructor instead of in the masterfulInstinctsStats here aswell